### ISSUE TITLE
Add more tags, sanitize existing

### ISSUE THREAD

**Issue by titpetric** [SlinSo/egon#1]
I would suggest adding a few tags from EJS:

```
<%= Outputs the value into the template (HTML escaped)
<%- Outputs the unescaped value into the template
<%# Comment tag, no execution, no output
%> Plain ending tag
-%> Trim-mode ('newline slurp') tag, trims following newline
```

The raw print block already exists as `<%==`, but I'd consider changing this to EJS syntax.

I'd also do something with `<%%` and `%%>`, changing the closing tag just to `%>`? With the addition of `-%>` (see above), we can get rid of some extra whitespaces because of `<%%` and `<%!` definitions.

**Comment by titpetric** [comment]
I added a quick support for '<%#' comment tags.
1. Used go fmt to fix styles
2. Renamed _block.go files to have block_ prefix (readability/sorting)
3. Added comments block

The way egon is laid out, makes it a bit tricky. If you're into it, we could discuss if there's some value to changing some egon internals - to have a stronger tokenizer which could parse this:

```
<%# this is a comment
 .... <% this is a code block within a comment %>
and this here is the end of the comment ->  %>
```

Currently, this is not possible as it doesn't at least count/match internal code blocks. This is also why I implemented `<%# ... #%>`. I couldn't submit a PR, maybe you have some additional options to check on the repo?

Best,
Tit
